_observing-your-data/query-insights/query-insights-dashboard.md

```diff
@@ -37,7 +37,7 @@ The **Top N queries** page provides a detailed overview of the queries that have
 
 The following image of the **Top N queries** page contains letter labels for each component.
 
-![Top N Queries Interface](../../images/Query-Insights/QueryInsights.png)
+<img src="{{site.url}}{{site.baseurl}}/images/Query-Insights/QueryInsights.png" alt="Top N Queries Interface">
```

Comment:
It should be correct to change the image URL like this, right?

Reply:
like this : https://opensearch.org/docs/latest/images/Query-Insights/QueryInsights.png

Reply:
@10000-ki Yes, this is correct, but usually we use Markdown syntax for uniformity.